### DISCUSSION TITLE
Disable long-press text selection & vibration

### DISCUSSION THREAD

**Discussion by merganon** [ionic-team/capacitor#5042] [Q&A]
When long-pressing anywhere, it will mark that text + vibrate the phone.

I managed to disable text selection by implementing this style:
```
html {
  -webkit-user-select: none;
  user-select: none;
}
```

But my phone is still vibrating. How do I disable that?

**Comment by mhrastegari** [comment]
Same issue

**Comment by mhrastegari** [comment]
Same issue
